doc/faq.rst

```diff
@@ -0,0 +1,191 @@
+by a random string.
+
+The :class:`autosklearn.classification.AutoSklearnClassifier` and all other *auto-sklearn*
+estimators accept the argument ``tmp_directory`` which change where such output is written to.
+
+There's an additional argument ``output_directory`` which can be passed to *auto-sklearn* and it
+controls where test predictions of the ensemble are stored if the test set is passed to ``fit()``.
+
+Auto-sklearn eats up all my disk space
+--------------------------------------
+
+*auto-sklearn* heavily uses the hard drive to store temporary data, models and log files which can
+be used to inspect the behavior auto Auto-sklearn. By default, *auto-sklearn* stores around 50
+models and their predictions on the validation data (which is a subset of the training data in
+case of holdout and the full training data in case of cross-validation) on the hard drive. Models
+are removed everytime the ensemble builder finishes an iteration, which means that the number of
+models stored on disk is only an approximation. One can therefore change the number of models
+that will be stored on disk by passing an integer for the argument ``max_models_on_disc``.
+
+As the number of models is only an indicator of the disk space used it is also possible to pass
+the memory in MB the models are allowed to use as a float. As above this is rather a guideline on
+how much memory is used as the models are removed from disk everytime the ensemble builder
+finishes an iteration. Especially when running in parallel it can happen that multiple models are
+constructed during one run of the ensemble builder.
+
+Available machine learning models
+=================================
+
+Will non-scikit-learn models be added to Auto-sklearn?
+------------------------------------------------------
+
+The short answer is unfortunately no.
+
+The long answer answer is a bit more nuanced: maintaining Auto-sklearn requires a lot of time and
+effort, which would grow even larger when depending on more libraries. Also, adding more
+libraries would require us to generate meta-data more often. Lastly, having more choices does not
+guarantee a better performance for most users as having more choices demands a longer search for
+good models and can lead to more overfitting.
+
+Nevertheless, everyone can still add his or her favorite model to Auto-sklearn's search space by
+following the `examples on how to extend Auto-sklearn
+<https://automl.github.io/auto-sklearn/master/examples/index.html#extension-examples>`_.
+
+If there is interest in creating a auto-sklearn-contrib repository with 3rd-party models please
+open an issue for that.
+
+Can the preprocessing be disabled
+---------------------------------
+
+No, but we're working on that.
+
+Usage
+=====
+
+Only use interpretable models
+-----------------------------
+
+Auto-sklearn can be restricted to only use interpretable models and preprocessing algorithms.
```

Comment:
Auto-sklearn can be restricted to only use a subset of all available models.

Reply:
Same as above.

Reply:
ok